soc/arm/st_stm32/stm32l4/power.c

```diff
@@ -8,6 +8,10 @@
 #include <soc.h>
 #include <init.h>
 
+#include <stm32_ll_rcc.h>
+#include <stm32_ll_utils.h>
```

Comment:
Why are these two includes needed here, but not in stm32wb/power.c? What's the difference?

Reply:
The reason is that the clock_stm32_ll_common_.h   includes the LL_RCC_MSI_Disable and LL_UTILS_PLLInitTypeDef
for the stm32l4 soc serie (which is not the case with the wb55)

Reply:
Sorry, but I don't see any includes at all in clock_stm32_ll_common.h.

Reply:
That's why they are in the power.c.

Reply:
Yes, but then how exactly does stm32wb/power.c work without them? Is it supposed to be this way? Though it's definitely not a blocker.

Reply:
It should work the the same as it works on L4 with PLL. But point is main BLE config doesn't use PLL, meaning PLL is not a likely config on WB.

Reply:
@erwango thanks for the clarification!
@FRASTM wouldn't it be more consistent to include <stm32l4xx_ll_utils.h> instead of the generic one?

Reply:
yes, changing to `stm32l4xx_ll_utils.h`

Reply:
Can someone explain why the non-generic libraries would be more consistent? (Since I also have opened #32322)

Reply:
> Can someone explain why the non-generic libraries would be more consistent? (Since I also have opened #32322)

My point was that all the other similar includes in soc.c files across the series are non-generic, so it would be better to do the same, or to change all of them to generic ones at once.

Reply:
Aaaah alright, thanks!